module/icp/algs/sha1/sha1.c

```diff
@@ -48,8 +48,8 @@ static void Encode(uint8_t *, const uint32_t *, size_t);
 #if defined(__sparc)
 
 #define	SHA1_TRANSFORM(ctx, in) \
-		SHA1Transform((ctx)->state[0], (ctx)->state[1], (ctx)->state[2], \
-				(ctx)->state[3], (ctx)->state[4], (ctx), (in))
+	SHA1Transform((ctx)->state[0], (ctx)->state[1], (ctx)->state[2], \
+		(ctx)->state[3], (ctx)->state[4], (ctx), (in))
```

Comment:
nit: It looks like this style fix should be in 3b05b504be9c4b845a5e6b563dcf38e026327498

Reply:
I've rebased the branch and that styling issue fix is now in its correct place.